src/ui/components/FormOverlay/index.js

```diff
@@ -104,52 +104,52 @@ export class FormOverlayBase extends React.Component<Props> {
         className={makeClassName('FormOverlay', className)}
         role="presentation"
       >
-        <div
+        <form
           onClick={this.onClickOverlay}
           className="FormOverlay-overlay"
           role="presentation"
         >
-          <div className="FormOverlay-close-control">
+          <header>
+            <div className="FormOverlay-close-control">
+              <Button
+                className="FormOverlay-close-button"
+                onClick={this.onClickExIcon}
+              >
+                <Icon name="x-mark" alt={i18n.gettext('Click to close')} />
+              </Button>
+            </div>
+            <h3 className="FormOverlay-h3">{title}</h3>
+          </header>
+          <section className="FormOverlay-content">
+            {children}
+          </section>
+          <footer className="FormOverlay-footer">
+            {/*
+              type=button is necessary to override the default
```

Comment:
Thanks, that's handy to have as a reference 👍 

Maybe we eventually want a `CancelButton` that does this for us, it seems common.